languages/csharp/exercises/concept/interfaces/.meta/Example.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+
+public interface IRemoteControlCar
+{
+    void Drive();
+    decimal DistanceTravelled { get; }
```

Comment:
`decimal` is usually only used in monetary domains IIRC. Maybe just just a double (or just a plain `int`) here?

Reply:
`int` - done.